### DISCUSSION TITLE
octoprint stops extruding after first layer

### DISCUSSION THREAD

**Discussion by orangoo** [Jyers/Marlin#1029] [Q&A] [answered]
Hello

I have Jyers firmware v1.3.2 installed. 
All prints fine via the micro USB card.

But when printing an object using Octoprint the print begins with the first layer and then during the second layer it stops extruding. I tested the same object via the micro USB card and that prints with no issues. (I did this to determine if I had a clogged nozzle). Performed one last test via Octoprint and the same problem occurs. Is there a setting I need to place in Octoprint to stop this issue.

I also imported the start and end GCODE from prusaslicer in case that fixed it.

Thanks

**Comment by Jyers** [comment, marked as answer]
I would recommend updating to the latest version to see if that fixes your issue first, then if not we can look further into the issue

**Reply by orangoo** [reply]
Thanks. Ended up reflashing the card for the RPI and works now!
